### DISCUSSION TITLE
Welcome to JoyStick Discussions!

### DISCUSSION THREAD

**Discussion by bobboteck** [bobboteck/JoyStick#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DickvdW** [comment]
At the moment I try to make a remote control for a model boat. I use a variant with 2 joysticks. The Webserver is running on an ESP32 and I want to pass the joy1X, joy1Y, joy2X and joy2Y values to the arduino programm with some kind of an event. I am trying all kinds of things to get an event but I was not succesfull up to know. Can you make a suggestion?

**Reply by bobboteck** [reply]
In your page you can use the callback function and send data to an API implemented on the ESP32, or similar solution, but I don't have tested before

**Comment by chepo92** [comment]
Hi @bobboteck , how are you doing? 
Are you still maintaining this repo ? I would like to contribute maintaining it if possible

**Reply by bobboteck** [reply]
I don't have enoght time to spend on this project, what you thing to do?

**Reply by chepo92** [reply]
Mainly improving code, merging some pull request that solve some issues (i faced the multi joystick issue). 
Addressing some reported issues, enchancements, etc.
I would do it in my own fork but it doesn't have the same attention and traffic as the original repo, that's why I first prefer to offer helping to maintain the repo.

**Reply by bobboteck** [reply]
Hi Alex, I add You as Collaborator, I hope you do a good work.
I try to activate the web page, for the repository, but at address [JoyStick](https://github.com/bobboteck/JoyStick) I see only the readme file, so I think that it need to have a index.html page, but I don't try it. You can try to change the file name to joy.html, and is good idea too redesign this page.
I would really appreciate it if you used a new Discussions thread to track your activities on the repository.
Thanks
